resources/scss/modules/_buttons.scss

```diff
@@ -4,29 +4,20 @@
     line-height: normal;
 
     &:not(.flat) {
-        border: 2px solid $color-accent-main-base;
-        color: $color-accent-main-base;
+        background-color: $color-accent-main-base;
+        color: $color-accent-main-text;
         border-radius: 24px;
         padding: $spacing-micro $spacing-small;
-
         text-align: center;
-
-        transition: background-color .1s ease-in-out;
+        transition: .3s ease-in-out;
     }
 
     &:not(.flat):hover {
-        background-color: $color-accent-main-base;
-        color: $color-accent-main-text;
+        box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
     }
 
     &.shop {
-        color: $color-accent-opt-base !important;
-        border-color: $color-accent-opt-base;
-    }
-
-    &.shop:hover {
         background-color: $color-accent-opt-base;
```

Comment:
The shop button lacks value contrast, it's hard to read.

![Screenshot from 2020-02-12 09-53-37](https://user-images.githubusercontent.com/12694995/74352049-94bebb80-4d7d-11ea-9339-a6021ffb5390.png)

We either need to change the `$color-accent-opt-base` variable to darken its background, but I tested and couldn't find a really satisfying tone. The other option is to use the dark text color for the text on the shop button: `$color-dark-base`. But the nav uses the #top-bar id right now, setting the text color to white. It'd involve removing the ID to allow overriding the styles I guess.

![Screenshot from 2020-02-12 09-54-05](https://user-images.githubusercontent.com/12694995/74352083-a0aa7d80-4d7d-11ea-8225-6087976dd135.png)